doc-content/enterprise-only/admin-and-config/kie-server-con.adoc

```diff
@@ -1,7 +1,7 @@
 [id='kie-server-con_{context}']
-= Red Hat {KIE_SERVER}
+= {KIE_SERVER}
 
-The Red Hat {KIE_SERVER} is the server where the rules and other artifacts for {PRODUCT} are stored. {KIE_SERVER} is a standalone, built-in component that can be used to instantiate and execute rules through interfaces available for REST, Java Message Service (JMS), or Java client-side applications,
+{KIE_SERVER} is the server where the rules and other artifacts for {PRODUCT} are stored. {KIE_SERVER} is a standalone, built-in component that can be used to instantiate and execute rules through interfaces available for REST, Java Message Service (JMS), or Java client-side applications,
```

Comment:
I'd say "are stored and executed" or "are stored and run" or something. It's not a storage server. Also not sure about the comma after "standalone".

Reply:
Done